docs/source/deployment/azure_functions.rst

```diff
@@ -0,0 +1,304 @@
+    }
+
+During Azure Functions initialized stage, it takes a while to download the docker image.
+Please wait few minutes before visiting your deployment.
+
+With the default authorization setting, your Azure Functions deployment is visible to
+all.  Open your browser and visit the URL in hostNames. You should see the web UI
+provided by BentoML API server.
+
+To test the prediction API in the Azure Functions deployment, you could use the web ui
+mentioned above or you could make a `curl` request to the endpoint.
+
+
+.. code-block:: bash
+
+    $  curl -i --request POST --header "Content-Type: application/json" \
+        --data '[[5.1, 3.5, 1.4, 0.2]]' \
+        "https://dev-azure-bentoml.azurewebsites.net/predict"
+
+    # Sample output
+
+    HTTP/1.1 200 OK
+    Content-Length: 3
+    Content-Type: application/json
+    Server: Kestrel
+    Request-Context: appId=cid-v1:c3a39ce6-5bf4-4961-a0de-01c0897b49de
+    request_id: 7a75f307-e62b-44f3-b834-7f87f9b96209
+    Date: Thu, 18 Jun 2020 20:06:01 GMT
+
+    [0]%
+
+
+Use `bentoml azure-functions list` to display all active deployments to Azure Functions
+
+.. code-block:: bash
+
+    $ bentoml azure-functions list
+
+    # Sample output
+    NAME           NAMESPACE    PLATFORM        BENTO_SERVICE                         STATUS    AGE
+    azure-bentoml  dev          azure-function  IrisClassifier:20200618124555_310CE0  running   19 minutes and 22.14 seconds
+
+
+
+==========================================
+Update existing Azure Functions deployment
+==========================================
+
+To update an active Azure Function deployment use `bentoml azure-functions update`
+command.
+
+.. code-block: bash
+
+    $ bentoml azure-functions update azure-bentoml -b IrisClassifier:new_version
+
+
+=================================
+Remove Azure Functions deployment
+=================================
+
+BentoML will remove all Azure resources created for the deployment.
+
+.. code-block:: bash
+
+    $ bentoml azure-functions delete azure-bentoml
+
+
+=====================================================================
+Deploy and manage Azure Functions deployment with remote YataiService
+=====================================================================
+
+BentoML recommends to use remote YataiService for managing and deploying BentoService
+when you are working in a team. To deploy Azure Functions in remote YataiService, you
+need to provide the Azure credential for it.
+
+In this guide, it will starts a docker container as remote YataiService. After Sign in
+with Azure CLI in your local machine, you should be able to find the `accessTokens.json`
+in your Azure directory. Now start the BentoML YataiService docker image and mount that
+`accessTokens.json` file to the running container.
+
+.. code-block:: bash
+
+    $ docker run -v /Users/bozhaoyu/.azure/accessTokens.json:/home/.azure/accessTokens.json -p 50051:50051 -p 3000:3000 bentoml/yatai-service:latest
```

Comment:
use `~` so user can copy and run: `docker run -v ~/.azure/accessTokens.json:/home/.azure/accessTokens.json -p 50051:50051 -p 3000:3000 bentoml/yatai-service:latest`

Reply:
Also, add an example of how to deploy it to a Kubernetes cluster? Adding secrete and mounting secret to the YataiServie container etc